pkg/konfluxgen/konfluxgen.go

```diff
@@ -50,6 +50,10 @@ func Generate(cfg Config) error {
 		return fmt.Errorf("failed to remove %q directory: %w", cfg.ResourcesOutputPath, err)
 	}
 
+	if err := os.RemoveAll(cfg.PipelinesOutputPath); err != nil {
+		return fmt.Errorf("failed to remove %q directory: %w", cfg.PipelinesOutputPath, err)
+	}
```

Comment:
Eventually, I believe we would leverage konflux auto upgrade of pipeline tasks, so I believe we would somehow "ignore changes"